src/components/curator-actions/fetch-signatories.ts

```diff
@@ -0,0 +1,64 @@
+import { get } from 'svelte/store';
+import { dotApi } from '../../stores';
+
+async function fetchCuratorProxyAddress(accountId: string): Promise<string | null> {
+	try {
+		const api = get(dotApi);
+		const proxy = await api.query.Proxy.Proxies.getValue(accountId);
+		const delegate = proxy[0][0].delegate;
+		return delegate;
```

Comment:
```suggestion
		return proxy[0][0].delegate;
```
Is the intermediate constant necessary?